### ISSUE TITLE
ftw() please

### ISSUE THREAD

**Issue by GReagle** [luaposix/luaposix#327]
Hello.  Would you consider adding ftw() or ntfw()?  They are documented at https://pubs.opengroup.org/onlinepubs/9699919799/functions/ftw.html 
and https://pubs.opengroup.org/onlinepubs/9699919799/functions/nftw.html

**Comment by gvvaughan** [comment]
Sure! Do you have a patch?

**Comment by GReagle** [comment]
I do not.  Which module would it belong in?

**Comment by gvvaughan** [comment]
I would say it needs its own ftw module, to follow the established pattern of lua modules named after posix headers.

**Comment by gvvaughan** [comment]
Actually, now that I read the SUSV3 I see that they have marked these APIs as obsolete.  Maybe a better approach is to implement the fts.h API calls in Lua over the existing luaposix wrapped calls?

Or at the very least to have a Lua implementation for ftw/nftw to fall back on when the host C library decided not to implement the obsolete calls.

**Comment by GReagle** [comment]
I think that nftw() is not obsolete.

**Comment by gvvaughan** [comment]
You’re right, thanks for checking. Let’s just add a thin wrapper for ntfw in a new ftw module then, and avoid all the Lua backfill code that would have been required for either of my suggestions above :-)

**Comment by cacarico** [comment]
I might take this one! Just looking for time to work on it, but since this issue opened for 5 years I don't think there's rush
